cdc/owner/changefeed.go

```diff
@@ -1045,13 +1048,23 @@ func (c *changefeed) checkUpstream() (skip bool, err error) {
 // observer, if needed run it in an independent goroutine with 5s timeout.
 func (c *changefeed) tickDownstreamObserver(ctx context.Context) {
 	if time.Since(c.observerLastTick.Load()) > downstreamObserverTickDuration {
+		c.observerLastTick.Store(time.Now())
+		select {
+		case <-ctx.Done():
+			return
+		default:
+		}
 		go func() {
 			cctx, cancel := context.WithTimeout(ctx, time.Second*5)
 			defer cancel()
 			if err := c.downstreamObserver.Tick(cctx); err != nil {
-				log.Error("backend observer tick error", zap.Error(err))
+				// Prometheus is not deployed, it happens in non production env.
+				if strings.Contains(err.Error(),
+					fmt.Sprintf(":%d", errno.ErrPrometheusAddrIsNotSet)) {
+					return
+				}
+				log.Warn("backend observer tick error", zap.Error(err))
```

Comment:
If the downstream is mysql, can the warn logs here be masked?

Reply:
If backend is MySQL, `downstreamObserver` is a dummy observer and err is always nil here.